css/templates/lune/pages/_product.css

```diff
@@ -1,3 +1,288 @@
 /* //////////////////////////////////////////////////////
    PRODUCT
-///////////////////////////////////////////////////////*/
\ No newline at end of file
+///////////////////////////////////////////////////////*/
+
+:root {
+  --hero-title-min-height: 238px;
+}
+
+.lune-product-section-hero {
+  min-height: calc(100vh - var(--navbar-height));
+}
+
+/* Set medium resolution make each section a minimum height of 100vh */
+@media (--breakpoint-md) {
+  .lune-product-section {
+    min-height: calc(100vh - var(--wide-navbar-height));
+  }
+}
+
+/* On some titles the titles are split on multiple lines. Apply a hyphen at the end. */
+.lune-title-has-mobile-hyphen:after {
+  content: '-';
+}
+
+.lune-page-product-wrapper {
+  background: inherit;
+}
+
+/* //////////////////////////////////////////////////////
+   HERO
+///////////////////////////////////////////////////////*/
+
+.lune-hero-content {
+  letter-spacing: var(--letter-spacing-3);
+  text-indent: var(--letter-spacing-3);
+  line-height: var(--line-height-2);
+  font-size: var(--h0);
+  font-weight: var(--font-weight-heavy);
+  margin-top: -var(--navbar-height);
+}
+
+.lune-hero-image-wrapper {
+  width: 100vmin;
+  min-width: 15rem;
+  max-width: 60vh;
+}
+
+.lune-hero-image {
+  margin-top: -0.5em;
+  max-width: 21.125rem; /* 338px */
+}
+
+.lune-hero-title {
+  font-size: inherit;
+  font-weight: inherit;
+  letter-spacing: inherit;
+  line-height: inherit;
+  text-indent: 0;
+  animation-delay: calc(var(--animation-delay-start) * 2);
+}
+
+.lune-hero-title-wrapper {
+  min-height: var(--hero-title-min-height);
+  margin-bottom: -1.9em;
+}
+
+@media (--breakpoint-md) {
+  .lune-hero-content {
+    font-size: 11.25rem; /* 180px */
+    margin-top: 0;
+    justify-content: flex-start;
+  }
+
+  .lune-hero-title {
+    display: inline;
+  }
+
+  .lune-title-has-mobile-hyphen:after {
+    content: none;
+    text-indent: inherit;
+  }
+
+  @media (--breakpoint-vld) {
+    .lune-hero-content {
+      letter-spacing: var(--letter-spacing-5);
+    }
+
+    .lune-hero-title:first-child {
+      text-indent: var(--letter-spacing-5);
+    }
+  }
+
+  /* On desktop the chair should be fixed to the viewport */
+  .lune-page-product .lune-hero-image-wrapper {
+    position: fixed;
+    top: calc(var(--wide-navbar-height) + (var(--hero-title-min-height) / 2));
+    left: 50%;
+    will-change: transform;
+    transform: translateX(-50%);
+  }
+
+  .lune-hero-image {
+    margin-top: 0;
+  }
+
+  .lune-hero-title-wrapper {
```

Comment:
This selector is sitting within a breakpoint and can't set the display property for a certain breakpoint with bass css